lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -214,6 +214,77 @@ speed:
   config_set_append: "speed %s"
   default_value: "auto"
 
+stp_bpdufilter:
+  kind: string
+  config_get_token_append: ['/^spanning-tree bpdufilter (.*)$/']
+  config_set_append: ["%s spanning-tree bpdufilter %s"]
+  default_value: false
+
+stp_bpduguard:
+  kind: string
+  config_get_token_append: ['/^spanning-tree bpduguard (.*)$/']
+  config_set_append: ["%s spanning-tree bpduguard %s"]
+  default_value: false
+
+stp_cost:
+  kind: string
+  config_get_token_append: ['/^spanning-tree cost (.*)$/']
+  config_set_append: ["spanning-tree cost %s"]
+  default_value: 'auto'
+
+stp_guard:
+  kind: string
+  config_get_token_append: ['/^spanning-tree guard (.*)$/']
+  config_set_append: ["%s spanning-tree guard %s"]
+  default_value: false
+
+stp_link_type:
+  kind: string
+  config_get_token_append: ['/^spanning-tree link-type (.*)$/']
+  config_set_append: ["spanning-tree link-type %s"]
+  default_value: 'auto'
+
+stp_mst_cost:
+  multiple:
+  config_get: "show running interface"
```

Comment:
For range based attributes, if we do "show running interface all" we get results for the ranges which are not set as well and this messes up the comparison. So use only "show running config interface" for them.